content/en/docs/concepts/containers/container-environment.md

```diff
@@ -39,9 +39,11 @@ as are any environment variables specified statically in the Docker image.
 
 ### Cluster information
 
-A list of all services that were running when a Container was created is available to that Container as environment variables.
-This list is limited to services within the same namespace as the new Container's Pod and Kubernetes control plane services.
-Those environment variables match the syntax of Docker links.
+When the [Pod spec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core) field `enableServiceLinks` is `true` (the default),
+a list of services are available to the Pod's Containers as environment variables.
+This list is limited to services within the same namespace as the new Container's Pod and Kubernetes control plane services, which existed at the time the Container was created.
+
+These environment variables match the syntax of Docker links.
```

Comment:
Let's explain this syntax. We can't assume that readers are familiar with Docker.